projects/packages/jetpack-mu-wpcom/src/features/wpcom-site-menu/wpcom-site-menu.php

```diff
@@ -575,3 +587,20 @@ function_exists( 'wpcom_site_has_feature' ) &&
 	}
 }
 add_action( 'admin_menu', 'wpcom_add_plugins_menu' );
+
+/**
+ * Hide the 'Help Center' icon in WordPress admin.
+ */
+function hide_help_center_admin_bar() {
+	if ( ! function_exists( 'is_agency_managed_site' ) || ! is_agency_managed_site() ) {
+		return;
+	}
+	?>
+	<style>
+		#wp-admin-bar-help-center {
+			display: none !important;
+		}
+	</style>
+	<?php
+}
+add_action( 'admin_head', 'hide_help_center_admin_bar' );
```

Comment:
This gets the job done but is a bit hacky. Open to improvements.

Hides Help Center icon:
<img src="https://private-user-images.githubusercontent.com/5634774/341844256-bf81ac2c-2ec5-48f7-83c6-9093a2d02dbd.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTkwMDk4MTgsIm5iZiI6MTcxOTAwOTUxOCwicGF0aCI6Ii81NjM0Nzc0LzM0MTg0NDI1Ni1iZjgxYWMyYy0yZWM1LTQ4ZjctODNjNi05MDkzYTJkMDJkYmQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDYyMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA2MjFUMjIzODM4WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MWMzMDczZmQxYzhlYjI1N2JmODczNzUyYTM1YzVjYmYxMmNlYzYyMzI4ZDgwZjU4YjBhNDFiN2ZjYzU4MmMwZiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.XZvlMvQcwM-sqq8jLre72ORQV-MmOzzlbq23Clax8Ao" />